MovieFirst/MovieFirst/Controllers/CurrentMovieViewController.swift

```diff
@@ -0,0 +1,448 @@
+// CurrentMovieViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// MARK: - CurrentMovieViewController
+
+/// Экран с выбранным фильмом
+final class CurrentMovieViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let overviewText = "Overview"
+        static let firstPartURLText = "https://api.themoviedb.org/3/movie/"
+        static let secondPartURLText = "/similar?api_key=8216e974d625f2a458a739c20007dcd6&language=ru-RU&page=1"
+        static let posterPathQueryText = "https://image.tmdb.org/t/p/w500"
+        static let systemPinkColorName = "SystemPinkColor"
+        static let releaseDataLabelText = "Релиз:"
+        static let voteAverageLabelText = "Оценка:"
+        static let voteCountLabelText = "Голоса:"
+        static let similarMovieLabelText = "Похожие фильмы"
+        static let similarMovieCollectionViewCellText = "SimilarMovieCollectionViewCell"
+    }
+
+    // MARK: - Private Visual Properties
+
+    private let imageMovieImageView: UIImageView = {
+        let imageView = UIImageView()
+        return imageView
+    }()
```

Comment:
```suggestion
    private let  movieImageView = UIImageView()
```
раз тут никаких свойств не устанавливается, то лучше писать так